Gemfile

```diff
@@ -11,38 +11,42 @@ def location_for(place, fake_version = nil)
-  gem 'metadata-json-lint',                                         :require => false
   gem 'rspec-puppet-facts',                                         :require => false
-  gem 'rspec',                                                      :require => false
-  gem 'puppet-blacksmith',                                          :require => false, :git => 'https://github.com/voxpupuli/puppet-blacksmith.git'
-  gem 'voxpupuli-release',                                          :require => false, :git => 'https://github.com/voxpupuli/voxpupuli-release-gem.git'
-  gem 'rubocop', '~> 0.38',                                         :require => false
   gem 'rspec-puppet-utils',                                         :require => false
-  gem 'puppetlabs_spec_helper',                                     :require => false
+  gem 'puppet-lint',                                                :require => false, :git => 'https://github.com/rodjek/puppet-lint.git'
   gem 'puppet-lint-absolute_classname-check',                       :require => false
   gem 'puppet-lint-leading_zero-check',                             :require => false
   gem 'puppet-lint-trailing_comma-check',                           :require => false
   gem 'puppet-lint-version_comparison-check',                       :require => false
   gem 'puppet-lint-classes_and_types_beginning_with_digits-check',  :require => false
   gem 'puppet-lint-unquoted_string-check',                          :require => false
   gem 'puppet-lint-variable_contains_upcase',                       :require => false
+  gem 'metadata-json-lint',                                         :require => false
+  gem 'puppet-blacksmith',                                          :require => false, :git => 'https://github.com/voxpupuli/puppet-blacksmith.git'
+  gem 'voxpupuli-release',                                          :require => false, :git => 'https://github.com/voxpupuli/voxpupuli-release-gem.git'
+  gem 'puppet-strings',                                             :require => false, :git => 'https://github.com/puppetlabs/puppetlabs-strings.git'
+  gem 'rubocop-rspec', '~> 1.5',                                    :require => false
 end
 
 group :development do
   gem 'travis',       :require => false
   gem 'travis-lint',  :require => false
   gem 'guard-rake',   :require => false
-  gem 'simplecov',    :require => false
-  gem 'listen', '<= 3.0.6', :require => false
 end
 
 group :system_tests do
-  gem 'winrm', '1.8.1'
-  gem 'beaker', '2.43.0'
-  gem 'beaker-rspec', '5.4.0'
+  gem 'beaker', '2.43.0',              :require => false
+  if beaker_version = ENV['BEAKER_VERSION']
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```

Comment:
I just noticed that our default Gemfile is a bit wrong:
```ruby
group :system_tests do
  gem 'beaker',                        :require => false
  if beaker_version = ENV['BEAKER_VERSION']
    gem 'beaker', *location_for(beaker_version)
  end 
  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
    gem 'beaker-rspec', *location_for(beaker_rspec_version)
  else
    gem 'beaker-rspec',  :require => false
  end 
  gem 'beaker-puppet_install_helper',  :require => false
end
```

we already include beaker twice if an ENV var is set. So we can stay with the changes in this PR?